meson.build

```diff
@@ -92,7 +92,10 @@ thread_dep = dependency('threads')
 #
 # Read build files from sub-directories
 #
-subdirs = [ 'lib', 'include', 'util', 'example', 'doc', 'test' ]
+subdirs = [ 'lib', 'include', 'example', 'doc', 'test' ]
+if not host_machine.system().startswith('freebsd')
```

Comment:
Are you sure this works? According to http://mesonbuild.com/Reference-manual.html#build_machine-object the test should be `host_machine.system() == 'bsd'`.